java/image.bzl

```diff
@@ -286,7 +286,7 @@ def _war_app_layer_impl(ctx):
   """Appends the app layer with all remaining runfiles."""
 
   available = depset()
-  for jar in ctx.attr.layers:
+  for jar in ctx.attr.jar_layers:
```

Comment:
I see, we do have layers.  This rename is better because these are an implementation detail.

Reply:
Ack.